libs/community/langchain_community/chat_models/minimax.py

```diff
@@ -77,9 +99,20 @@ def _convert_message_to_dict(message: BaseMessage) -> Dict[str, Any]:
     if isinstance(message, HumanMessage):
         message_dict = {"role": "user", "content": message.content}
     elif isinstance(message, AIMessage):
-        message_dict = {"role": "assistant", "content": message.content}
+        message_dict = {
+            "role": "assistant",
+            "content": message.content,
+            "tool_calls": message.additional_kwargs.get("tool_calls"),
```

Comment:
double-checking that passing `"tool_calls": None` on most messages won't break anything